src/test/java/modules/Register.java

```diff
@@ -21,24 +21,19 @@
     Date date = new Date();
     DateFormat dateFormat = new SimpleDateFormat("yyyy-mm-dd-HH-mm-ss");
 
-    @Given("^I'm in WestMarine Homepage$")
-    public void iMInWestMarineHomepage() throws Throwable {
-        browser.navigate().to(BASE_URL);
-    }
 
     @And("^I click to sign in$")
     public void iClickToSignIn() throws Throwable {
         browser.findElement(By.linkText("Sign In")).click();
-
     }
 
-    @And("^I have to choose create option$")
+    @And("^I choose the create option$")
     public void iHaveToChooseCreateOption() throws Throwable {
         sleep(2000);
```

Comment:
No sleeps anymore, handle this with implicit or explicit waits according to the case